### PR TITLE
Add default timeouts to waitForElementByXXX

### PR DESCRIPTION
**Pull request by lag-of-death**

### CODE REVIEW

**Review by ysukhoverkhov — APPROVED** [review]
Awesome, thanks :). I also will request Ivan or Ilya to check, they were a little bit in subject.

File: lib/commands.js [hunk 1]
```diff
@@ -852,7 +852,8 @@ commands.waitFor = function(){
   }
 
   // default
-  opts.timeout = opts.timeout || 1000;
+  opts.timeout = opts.timeout || config.getDOMConfig().timeout;
+
   opts.pollFreq = opts.pollFreq || 200;
```
Comment (ilya-manin) [inline lib/commands.js:857]: Maybe we could add also a configuration for polling frequency? However I don't remember that we configure this settings at cbt 🤔 @ysukhoverkhov what do you say on this?

**Review by ilya-manin — APPROVED** [review]
LGTM 👍

**Comment by lag-of-death** [comment]
To be able to test with `gulp test`, first do:
1. `npm i`
2. **`copy node_modules/sv-selenium/bin/setup.sh to node_modules/.bin/`**
3. `node_modules/.bin/install_selenium`
4. `node_modules/.bin/install_chromedriver` 
5. `node_modules/.bin/start_selenium_with_chromedriver`

Also, Firefox version 47.0.1 is needed, I got mine from https://ftp.mozilla.org/pub/firefox/releases/47.0.1/linux-x86_64/en-GB/firefox-47.0.1.tar.bz2
and symlinked like this: `ln -s /path/to/firefox-executable /usr/bin/firefox`

**Review by ipuchkov — APPROVED** [review]

**Pull request merged**